control_msgs/action/AntipodalGripperCommand.action

```diff
@@ -0,0 +1,11 @@
+sensor_msgs/JointState command # The velocity and effort fields are optionally and not used if empty.
+---
+float64 position  # The current gripper gap size (in meters)
+float64 effort    # The current effort exerted (in Newtons)
+bool stalled      # True iff the gripper is exerting max effort and not moving
+bool reached_goal # True iff the gripper position has reached the commanded setpoint
```

Comment:
Is there value in reporting `reached_goal` or `stalled` in the feedback topic? If the feedback was a `JointState` the caller could inspect the joint velocity and I would suggest changing `reached_goal` to something like percent complete over a bool.

Reply:
I am not sure why those are here. I can actually understand stalled because some users might want to monitor stalled and send a cancel goal after it has been stalled too long. But I don't know why the goal term would be in the feedback.

Reply:
I vote we remove both of these options from the feedback topic.